engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -318,7 +319,8 @@ public static void main(String[] args) throws Throwable
       logger.error("Fatal {} in container!", (e instanceof Error) ? "Error" : "Exception", e);
       /* Report back any failures, for diagnostic purposes */
       try {
-        umbilical.reportError(childId, null, ExceptionUtils.getStackTrace(e));
+        LogFileInformation logFileInfo = LoggerUtil.getLogFileInformation();
```

Comment:
LoggerUtil.getFileInformation does not check the concrete logger instance level, it checks FileAppender level.